modules/understanding-agent-install.adoc

```diff
@@ -66,5 +66,9 @@ The following platforms are supported:
 +
 [IMPORTANT]
 ====
-The `none` option requires the provision of DNS name resolution and load balancing infrastructure in your cluster. See the _Requirements for a cluster using the platform "none" option_ section for more information.
+For platform `none`:
+
+* The `none` option requires the provision of DNS name resolution and load balancing infrastructure in your cluster. See the _Requirements for a cluster using the platform "none" option_ section for more information.
```

Comment:
In a case where the _Requirements for a cluster using the platform "none" option_ section itself is simply further down the same assembly this module is in, is it still worth pointing users to the additional resources under this current module? If so, I can go ahead and mention additional resources, I just don't know whether it would be weird to mention it in this kind of case.

Reply:
I mentioned it because while reading, I started browsing the side pane for that particular section before noticing that it was in the additional resources. :slightly_smiling_face:

Reply:
It's entirely up to you, of course, that's just something that I noticed.

Reply:
I suppose since the additional resources link is supposed to be a convenience anyways, I can go ahead and call it out:
```suggestion
* The `none` option requires the provision of DNS name resolution and load balancing infrastructure in your cluster. See _Requirements for a cluster using the platform "none" option_ in the "Additional resources" section for more information.
```